catalog/pipeline/fbc/0.1/fbc.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: bundle-fbc-pipelinerun
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: fbc
+spec:
+  description: >-
+    Tekton pipeline to release HACBS Application Snapshot to Quay
+  params:
+    - name: applicationSnapshot
+      type: string
+      description: The ApplicationSnapshot in JSON format
+    - name: enterpriseContractPolicy
+      type: string
+      description: JSON representation of the EnterpriseContractPolicy
+    - name: iibJsonBuildArgs
+      type: string
+      description: IIB JSON Arguments to add a new build
+    - name: iibUrl
+      type: string
+      description: IIB Service URL address
+    - name: iibSecret
+      type: string
+      description: Secret containing the credentials for IIB service
+  workspaces:
+    - name: release-workspace
+  tasks:
+    - name: verify-enterprise-contract
+      taskRef:
+        name: verify-enterprise-contract-v2
+        bundle: quay.io/hacbs-release/verify-enterprise-contract-v2:main
+      params:
+        - name: IMAGES
+          value: $(params.applicationSnapshot)
+        - name: SSL_CERT_DIR
+          value: /var/run/secrets/kubernetes.io/serviceaccount
+        - name: POLICY_CONFIGURATION
+          value: $(params.enterpriseContractPolicy)
+        - name: STRICT
+          value: "0"
+    - name: add-bundles-to-image-index
+      taskRef:
+        name: add-bundles-to-image-index
+        bundle: quay.io/hacbs-release/add-bundles-to-image-index:main
```

Comment:
shouldn't this be version 0.1, not main?

Reply:
That's a good question actually. For the pipeline `release` it uses `main` in the pipeline file and `0.X` in the `sample` and `run` files. I just did the same for this one. Not sure which one is right.

Reply:
Looking into `quay.io/hacbs-release` the right should be or `main` or the SHA256 tag.

Reply:
But isn't this saying what version of the add-bundles-to-index-image task the 0.1 fbc pipeline should use? Main and 0.1 for the task will be the same as it is the first one, but once we have a 0.2 version (which will then be main) of add-bundles-to-index-image, if we, for example, add a required parameter to the task, won't this 0.1 fbc pipeline then be broken?

Reply:
I think I found what you mean. Just noticed also that in the repo we have for instance `skopeo-copy` and `task-skopeo-copy`.  One has `main` and the second one has `0.1`. I guess the second is product from the automated build. Nice that you mentioned. I probably might need to rename some places to use `task-bundles-to-image-index` instead, so it can reach the image.

Reply:
ahh yes, I think with the automation we add the prefix to make it more clear if it is a pipeline or task in case the name is ambiguous. So yeah, you will need to change to `task-add-bundles-to-image-index` I believe